### DISCUSSION TITLE
ch6-cloud/ch6-03-delay-job

### DISCUSSION THREAD

**Discussion by giscus** [chai2010/advanced-go-programming-book#640] [General]
# ch6-cloud/ch6-03-delay-job

Go语言高级编程

https://chai2010.cn/advanced-go-programming-book/ch6-cloud/ch6-03-delay-job.html

**Comment by gocpplua** [comment]
图 6-8 所示 2 只有一个副本， 3 有三个副本。 这个是不是有误？ 应该2 3 都只有两个副本
